lib/fakefs/base.rb

```diff
@@ -15,13 +15,27 @@ def Pathname(*args)
 
 # FakeFS module
 module FakeFS
+  class ActivationError < StandardError
+    def initialize(current_state, required_state)
+      states = current_state ? %i[without with] : %i[with without]
```

Comment:
this might be more readable:
```suggestion
      requested, current = (current_state ? %i[without with] : %i[with without])
```

Reply:
Indeed - done ✅